tutorials/iot-device-to-device/index.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Google Cloud IoT Core Device to Device Communication
```

Comment:
Perhaps start the title with an action, such as Configure Google Cloud IoT Core Device to Device Communication